crates/domain-model/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+authors = [
+    "Valentin Tolmer <valentin@tolmer.fr>",
+    "Simon Broeng Jensen <sbj@cwconsult.dk>",
```

Comment:
Oh, also: for the validation crate, I was happy to have you as an author, since, you know, you wrote the code. 
For this crate I think you're not there yet :)

Reply:
Agreed, sorry, it's a copy+paste leftover. :-)